Tasks/DeployAzureResourceGroup/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,6 +1,6 @@
 {
   "loc.friendlyName": "Azure Resource Group Deployment",
-  "loc.helpMarkDown": "[More Information](https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/DeployAzureResourceGroup/README.md)",
+  "loc.helpMarkDown": "[More Information](https://aka.ms/argtaskreadme)",
```

Comment:
Should we rather not have the link to v1 documentation. Talk to Usha to see which one should we have. Also you need to update the task version